master/painting.html

```diff
@@ -2361,7 +2361,7 @@ <h3 id="MarkerElement">The <span class="element-name">'marker'</span> element</h
   <tr>
     <td><dfn id="MarkerWidthAttribute" data-dfn-type="element-attr" data-dfn-for="marker" data-dfn-export="">markerWidth</dfn>,
     <dfn id="MarkerHeightAttribute" data-dfn-type="element-attr" data-dfn-for="marker" data-dfn-export="">markerHeight</dfn></td>
-    <td><a>&lt;length&gt;</a> | <a>&lt;percentage&gt;</a> | <a>&lt;number&gt;</a></td>
+    <td><a>&lt;length-percentage&gt;</a> | <a>&lt;number&gt;</a></td>
```

Comment:
are these ones that use length, number, and percentage correct to use length-percentage (rather than say number-percentage)? As far as I understand it you use length-percentage if percentage computes to length.

Reply:
Yes, in most places the length | number | percentage SVG properties are all lengths, with the numbers computing to lengths as "SVG user units". Allowing numbers without units in the CSS syntax is a legacy feature, we don't need to allow them in `calc()` combinations.